api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -126,13 +117,11 @@
       NotebooksService notebooksService,
       Provider<DbUser> userProvider,
       @Qualifier(DatasetConfig.DATASET_PREFIX_CODE) Provider<String> prefixProvider,
-      WorkspaceService workspaceService,
-      ConceptSetMapper conceptSetMapper) {
```

Comment:
Ah. You're moving the usage of the mapper, not the mapper itself. I was confused for a second.